wiki/Skinning/Tutorial/en.md

```diff
@@ -1,105 +1,136 @@
+
+- **Trim or crop** your images whenever possible. The osu!client will render every pixel of an image which will result in a bigger workload!
+- **Compress** images whenever possible. Compressing usually removes unnecessary info about blank pixels and reduces the file sizes dramatically.
+- Understand that certain elements **face a certain direction.** Some elements have a standard as to which direction they must face to face the correct direction when the osu!client uses it.
+
+### What are overlays?
+
+Overlays will *overlay* an element's image **and will not be colored or tinted**. Note that only a few elements contains an overlay.
+
+### Resolutions
+
+Because _osu!_ can be played at different resolutions, some skinning elements may overlap or be farther apart than what is expected. Meaning that not elements will scale themselves to fit the game window resolution.
+
+It is best to keep the following resolutions in mind while skinning.
+
+- 1024 x 768 (4:3, standard game resolution, the game is based on it)
+- 2048 x 1536 (4:3, standard game resolution in HD scaling)
+- 1366 x 768 (16:9, standard game widescreen resolution)
+- 2732 x 1536 (16:9, standard game widescreen resolution in HD scaling)
+- 1920 x 1080 (16:9, standard HD resolution)
+
+Images will be adjusted by the game itself to fit resolutions derivating from the ones mentioned above. Most of them will be rescaled to fit the playfield or repositioned on different aspect ratios.
+
+#### HD Sprites
+
+HD sprites are used in place of the normal sprites on higher resolutions, when present. Compared to normal sprites, HD sprites look much sharper and cleaner on these larger resolutions. They get scaled by the game itself automatically to fit the resolution used. Resolutions supporting HD sprites begin at a **minimum of 800 pixel in height**. HD sprites are tagged with `@2x` at the end of their names. For example: `cursor@2x.png`.
+
+HD sprites have doubled dimension sizes. For example: the normal `hitcircle.png` sprite has the dimension sizes of **128x128 pixels.** Where its HD sprite, `hitcircle@2x.png`, has the dimension sizes of **256x256 pixels.**
+
+Every sprite has an HD counterpart, even all frames in an animation can have HD counterparts. As a result, the filesize of the folder or archive will increase due to having more images than normal.
+
+All HD sprites may also be bigger in filesize due to the fact that the canvas size used is four times bigger compared to the normal sprite. There are essentially two resolution modes _osu!_ is using. Each of them prefers one set of sprites. The first mode is **LowResolution** while the second mode is **HighResolution.**
+
+- *LowResolution mode* uses the normal sprites and ignores the HD sprites **(SD-resolution skin)**
+- *HighResolution mode* prefers HD sprites and uses normal sprites as a fallback if no HD sprite is available **(HD-resolution skin)**
+
 ### Can someone make this skin from that show/game?
```

Comment:
Can someone make a skin about that universe?

Reply:
I'm going to keep it as `that show/game`, I'm not sure if it is common knowledge that "universe" could mean a sandbox where characters/ideas cease to exist 🤷‍♂️